templates/fine-tune-llm_v2/cookbooks/modifying_hyperparameters/README.ipynb

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
Also include num_epochs, talk about validation at each epoch, talk about controlling the checkpointing every epoch and num_keep, etc.